### PR TITLE
feat: prevent modifying DESTROYING status services

### LINKED ISSUE #2195: Let's prevent modifying `DESTROYING` status services.
### What Operating System(s) are you seeing this problem on?

Linux (x86-64)

### Backend.AI version

24.03

### Describe the bug

Even if the service status is `DESTROYING`, I can modify or generate token using that service.
Let's prevent modifying `DESTROYING` status services.

### To Reproduce

1. Go to Serving page.
2. Make a new service
3. Remove it
4. When the service status is `DESTROYING`, edit that service or click the generate token.
5. See Error.

### Expected Behavior

Disable edit, generate token button if the status is `DESTROYING`.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ironAiken2**
### This PR resolves #2195 
feature:
- Gets the status data and uses it for the button disable property.

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: react/src/pages/RoutingListPage.tsx [hunk 1]
```diff
@@ -265,7 +266,10 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
           <Button
             type="primary"
             icon={<SettingOutlined />}
-            disabled={(endpoint?.desired_session_count || 0) < 0}
+            disabled={
+              (endpoint?.desired_session_count || 0) < 0 ||
```
Comment (agatha197) [inline react/src/pages/RoutingListPage.tsx:270]: ~~User can modify `desired_session_count` later. Leave just filtering the status.~~

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Comment by yomybaby** [comment]
@ironAiken2 Please don't forget to add the Frontend daily board project when you do something.

File: react/src/pages/RoutingListPage.tsx (hunk 1 above)
Comment (ironAiken2) [reply react/src/pages/RoutingListPage.tsx:270]: @agatha197 I just removed desired_session_count.

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**